runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -346,6 +358,11 @@ private void addSubscriptions()
 		eventBus.subscribe(MenuOpened.class, this, this::onMenuOpened);
 		eventBus.subscribe(MenuEntryAdded.class, this, this::onMenuEntryAdded);
 		eventBus.subscribe(FocusChanged.class, this, this::onFocusChanged);
+
+		if (config.lastJewel())
+		{
+			eventBus.subscribe(MenuOptionClicked.class, "duality and cruelty of poultry jewellery", this::onMenuOptionClicked);
```

Comment:
"duality and cruelty of poultry jewellery" should be a private static final String in global args